shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -566,7 +566,9 @@
       // All the children of this element are absolutely positioned and then
       // transformed, so manually set a height for the convenience of
       // embedders.
-      this.element.style.height = this.layout.offsetY + 'px';
+      var padding =
```

Comment:
We explicitly size the element, getBoundingClientRect would just return whatever height we last set (we could unset and re-set it, but then it'd just return the min-height + padding, which isn't what we want). This will reliably return the style's padding-bottom... I don't really like it, but short of introducing an explicit bottom-padding function to GaiaGrid, I'm not sure the best way of going about this :(